lib/jira/jwt_client.rb

```diff
@@ -14,6 +15,30 @@ def make_multipart_request(url, data, headers = {})
       super(url, data, headers)
     end
 
+    class JwtBuilder
```

Comment:
If `build_jwt` is the only user of jwt building do we need a class for this? Can we just 

```ruby
 claim = Atlassian::Jwt.build_claims \
          issuer,
          request_url,
          http_method.to_s,
          site,
          (Time.now - 60).to_i,
          (Time.now + 86_400).to_i

        JWT.encode claim, shared_secret
```

in the `build_jwt`?